task-definition-b2-backup.tftpl

```diff
@@ -0,0 +1,108 @@
+[
+  {
+    "dnsSearchDomains": null,
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "${cw_log_group}",
+        "awslogs-region": "${aws_region}",
+        "awslogs-stream-prefix": "${cw_stream_prefix}"
+      }
+    },
+    "entryPoint": null,
+    "portMappings": [],
+    "command": [],
+    "linuxParameters": null,
+    "cpu": ${cpu},
+    "environment":
+      {
+        "name": "ATLAS_TOKEN",
+        "value": "${atlas_token}"
+      },
+      {
+        "name": "B2_ACCOUNT_ID",
+        "value": "${b2_access_key}"
+      },
+      {
+        "name": "B2_ACCOUNT_KEY",
+        "value": "${b2_secret_key}"
+      },
+      {
+        "name": "FSBACKUP_MODE",
+        "value": "${fsbackup_mode}"
+      },
+      {
+        "name": "ORGANIZATION",
+        "value": "${tfc_organization_name}"
+      },
+      {
+        "name": "RESTIC_BACKUP_ARGS",
+        "value": "${backup_args}"
+      },
+      {
+        "name": "RESTIC_FORGET_ARGS",
+        "value": "${forget_args}"
+      },
+      {
+        "name": "RESTIC_HOST",
+        "value": "${restic_host}"
+      },
+      {
+        "name": "RESTIC_PASSWORD",
+        "value": "${repo_pw}"
+      },
+      {
+        "name": "RESTIC_REPOSITORY",
+        "value": "${repo_string}"
+      },
+      {
+        "name": "RESTIC_TAG",
+        "value": "${restic_tag}"
+      },
+      {
+        "name": "SOURCE_PATH",
+        "value": "${source_path}"
+      },
+      {
+        "name": "AWS_REGION",
+        "value": "${aws_region}"
+      },
+      {
+        "name": "AWS_ACCESS_KEY_ID",
+        "value": "${aws_access_key_id}"
+      },
+      {
+        "name": "AWS_SECRET_ACCESS_KEY",
+        "value": "${aws_secret_access_key}"
+      },
+      {
+        "name": "LOG_LEVEL",
```

Comment:
Is this needed? I don't see it in tfc-backup-b2.

Reply:
I don't think it is needed.  It was a copy/paste artifact.